manifest/armv7l/l/libfontenc.filelist

```diff
@@ -1,7 +1,5 @@
-# Total size: 151382
+# Total size: 23583
```

Comment:
This value is a lot lower than the previous.  Calculation correct?

Reply:
No Longer in this PR.